api/falcon/v1alpha1/falconimageanalyzer_types.go

```diff
@@ -135,6 +143,37 @@ type FalconImageAnalyzerUpdateStrategy struct {
 	RollingUpdate appsv1.RollingUpdateDeployment `json:"rollingUpdate,omitempty"`
 }
 
+type FalconImageAnalyzerAgentServiceSpec struct {
+	// HTTP port for the IAR Agent Service.
+	// +kubebuilder:default:=8001
+	// +operator-sdk:csv:customresourcedefinitions:type=spec,displayName="IAR Agent Service HTTP Port",order=1
+	HTTPPort int32 `json:"httpPort,omitempty"`
+
+	// Auto update the certificates every time there is an update.
+	// +kubebuilder:default:=true
+	// +operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Auto Certificate Update",order=2
+	AutoCertificateUpdate bool `json:"autoCertificateUpdate,omitempty"`
```

Comment:
This is not referenced anywhere.  If we were to use it we could have `reconcileIARTLSSecret` check validity of the cert then if it's expired recreate it.  I think that would also need to also trigger a rolling restart of the deployment.  We could also exclude this for now, if needed.